public/index.html

```diff
@@ -2,12 +2,12 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <meta property="og:url"           content="%PUBLIC_URL%" />
+    <meta property="og:url"           content="%PUBLIC_URL%/" />
```

Comment:
I thought that it worked if you gave it absolute URLs, but might be mistaken
i.e. setting `"/"` as opposed to `"./"`

Reply:
Seems to want fully qualified urls